terraform/modules/eks/security.tf

```diff
@@ -60,8 +42,29 @@ resource "aws_security_group" "lb_sg" {
     protocol    = "-1"
     cidr_blocks = ["0.0.0.0/0"]
   }
+}
 
-    tags = {
-    Name = "${var.name_prefix}-lb-sg"
```

Comment:
Can you keep this tag in? so the security group name in console isn't blank. Thanks.